lib/builder/webpack/base.js

```diff
@@ -139,10 +139,25 @@ export default class WebpackBaseConfig {
       },
       {
         test: /\.jsx?$/,
-        exclude: file => (
-          /node_modules/.test(file) &&
-          !/\.vue\.js/.test(file)
-        ),
+        exclude: file => {
+          // not exclude files outside node_modules
+          if (/node_modules/.test(file)) {
+            let transpile = this.options.build.transpile || []
```

Comment:
Can be simplified to `[].concat(this.options.build.transpile)` (Supports both array and string :D)

Reply:
Good idear